explorer/frontend/scripts/docker/startup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file is not used in the Dockerfile, should it be removed?

Reply:
Removed.